### ISSUE TITLE
Add fga to Nix packages repository

### ISSUE THREAD

**Issue by mjarosie** [openfga/openfga#1048]
**Summary**

Nix is a purely functional package manager and deployment system for POSIX ([example introduction](https://nixos.org/guides/nix-pills/why-you-should-give-it-a-try#why-you-should-give-it-a-try)).

That'd be great if `fga` could be added to the package registry: https://nixos.org/manual/nixpkgs/stable/#chap-quick-start

**Comment by jon-whit** [comment]
@mjarosie want to contribute it? We'd love your help with the contribution.

**Comment by mjarosie** [comment]
Sorry I didn't have time to take a look at it yet. Hopefully soon!

**Comment by gepbird** [comment]
This was done in https://github.com/NixOS/nixpkgs/pull/310534

**Comment by aaguiarz** [comment]
@gepbird yes, however,[ I only see v1.5.5](https://search.nixos.org/packages?channel=24.05&show=openfga&from=0&size=50&sort=relevance&type=packages&query=openfga).

Do we need to add action to automate this?

**Comment by gepbird** [comment]
> @gepbird yes, however,[ I only see v1.5.5](https://search.nixos.org/packages?channel=24.05&show=openfga&from=0&size=50&sort=relevance&type=packages&query=openfga).
> 
> Do we need to add action to automate this?

1.5.5 is on the latest stable channel 24.05, where package updates are usually not backported.
You can click on unstable or use [this link](https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=openfga) to see that the latest version is 1.6.1.

If a project builds with the same nix file (that contains the dependencies and configuration for a package), then a bot will automatically open a package update PR, but a human still needs to review it to make sure there's no major regressions.
